cypress/support/helper.js

```diff
@@ -1,17 +1,14 @@
 import { knobsTab } from "../locators/commonLocators"
-import { draggableRecordByText } from "../locators/draggableContextLocators"
 import { DEBUG_FLAG } from ".";
 
 export function visitComponentUrl(component, suffix = 'default', iFrameOnly = false) {
     cy.visit(prepareUrl(component, suffix, iFrameOnly))
-    if(!iFrameOnly) knobsTab().click()
+    if (!iFrameOnly) knobsTab().click()
 }
 
-export function dragAndDropForDraggableRecord(record, position) {
+export function dragAndDrop(draggableRecord, destinationPosition, startFromHight) {
```

Comment:
Consider renaming record to element.

It would be good if this function could drag on the x axis also.

Perhaps we could have it accept a sequence of commands to guide the drag, so that we can use it for more complex navigation also.

Reply:
fixed